packages/miniflare/src/plugins/core/services.ts

```diff
@@ -90,6 +90,7 @@ export const ServiceDesignatorSchema = z.union([
 	z.object({
 		name: z.union([z.string(), z.literal(kCurrentWorker)]),
 		entrypoint: z.ostring(),
+		props: z.record(z.any()).optional(),
```

Comment:
Nit, but this might be better:
```suggestion
		props: z.record(z.unknown()).optional(),
```